dockerfiles/Dockerfile.arm32v7

```diff
@@ -1,5 +1,8 @@
 FROM balenalib/raspberrypi3-python:latest-stretch-build
 
+ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
+ARG ONNXRUNTIME_SERVER_BRANCH=master
```

Comment:
I don't think you need this here.

Reply:
The intent here, as suggested by an earlier review comment from @jywu-msft (both on this PR and PR #922), is to make all these Dockerfiles more consistent by having the same approach and build arguments in each.
https://github.com/microsoft/onnxruntime/pull/1222#issuecomment-501781216

The `ONNXRUNTIME_SEREVER_BRANCH` variable is used in the `git clone` command below at line 43.